cloud_functions/token_refresh/streams/strava.py

```
@@ -82,13 +83,16 @@ def strava_update_user_tokens(
+    )
+    strava_ref.update(
         {
-            "stream.strava.accessToken": new_tokens["accessToken"],
-            "stream.strava.refreshToken": new_tokens["refreshToken"],
-            "stream.strava.expiresAt": new_tokens["expiresAt"],
-            "stream.strava.lastTokenRefresh": new_tokens["lastTokenRefresh"],
+            "accessToken": new_tokens["accessToken"],
+            "refreshToken": new_tokens["refreshToken"],
+            "expiresAt": new_tokens["expiresAt"],
+            "lastTokenRefresh": new_tokens["lastTokenRefresh"],
         }
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)